source/core/src/main/com/csse3200/game/GdxGame.java

```diff
@@ -25,8 +26,11 @@ public void create() {
     logger.info("Creating game");
     loadSettings();
 
+    // Register the game instance with ServiceLocator
+    ServiceLocator.registerGame(this);
```

Comment:
Remove this - the game is not a service. See comments in MainMenuDisplay and ServiceLocator for more details.